schema.ts

```diff
@@ -1245,6 +1246,18 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * @hidden
    */
   groupIdProperty?: PropertyIdentifier<K>;
+
+  /**
+   * The name of the property within {@link ObjectSchemaDefinition.properties} that can be be interpreted as
+   * a large blob of unstructured content.
```

Comment:
I'm not sure it will be feasible to return a page of N large blobs of text within the time and memory limits of the Pack for all but very small values of N. For example. I wonder if like fetching permissions, this too should be a per-row function.

Reply:
@patrick-codaio Any thoughts. I know we discussed this as well. I do like that this is much much less work on our part but I see @ekoleda-codaio's points

Reply:
Definitely worth a discussion. The way I see things, there's not a clear boundary between something like a Jira issue description and a google doc. They're both big rich text blobs that we want Brain to understand similarly, and if they're not too big, it would be nice to have them in a doc. So I feel it's still appropriate to let them be "normal" fields.

Reply:
Discussed f2f. We will begin with this approach and add on the function approach as needed. They are not incompatible